docs/source/developer_guides/method_comparison/bone.md

```diff
@@ -0,0 +1,195 @@
+config = BoneConfig(
+    bottleneck_size=128,  # larger bottleneck
+    target_modules=["attention.output", "intermediate"],
+    dropout=0.2,
+    use_orthogonal=True,  # enable orthogonal transformations
+    orthogonal_eps=1e-6,  # epsilon for numerical stability
+)
+```
+
+## Hyperparameter Tuning
+
+### Recommended Ranges
+| Parameter | Recommended Range | Impact |
+|-----------|------------------|--------|
+| bottleneck_size | 32-256 | Larger = better performance, more parameters |
+| dropout | 0.0-0.3 | Regularization |
+| orthogonal_eps | 1e-8 to 1e-4 | Numerical stability |
+
+### Optimal Settings by Model Size
+| Model Size | Bottleneck Size | Dropout | Orthogonal Eps |
+|------------|----------------|---------|----------------|
+| < 100M    | 32            | 0.1     | 1e-6          |
+| 100M-1B   | 64            | 0.15    | 1e-6          |
+| 1B-7B     | 128           | 0.2     | 1e-5          |
+| 7B-13B    | 256           | 0.25    | 1e-5          |
+
+## Comparison with Other Methods
+
+### Performance Comparison
+| Method | Memory Efficiency | Training Speed | Model Size Suitability |
+|--------|------------------|----------------|-----------------------|
+| Bone   | Very High       | Fast          | Small-Medium         |
+| LoRA   | High            | Fast          | All                  |
+| Adapter | Medium         | Medium        | All                  |
+| Prompt | Very High      | Very Fast     | All                  |
+
+### Memory Usage Comparison
+| Method | Parameters (% of base) | Training Memory | Inference Memory |
+|--------|----------------------|-----------------|------------------|
+| Bone   | 0.05%               | Very Low       | Very Low         |
+| LoRA   | 0.1%                | Low            | Low              |
+| Adapter | 0.5%                | Medium         | Medium           |
+| Prompt | 0.01%               | Very Low       | Very Low         |
+
+## Best Practices
+
+1. **Bottleneck Size Selection**
+   - Start with size 64 for most cases
+   - Increase for better performance
+   - Consider model size and task complexity
+
+2. **Target Modules**
+   - Focus on attention outputs
+   - Add intermediate layers for complex tasks
+   - Consider model architecture
+
+3. **Training Tips**
+   - Use learning rate 5e-5 to 2e-4
+   - Monitor orthogonal condition
+   - Use gradient clipping
+
+## Common Issues and Solutions
+
+### Problem: Orthogonal Instability
+**Solution:**
+```python
+# Improve numerical stability
+config = BoneConfig(
+    bottleneck_size=64,
+    target_modules=["attention.output"],
+    dropout=0.1,
+    use_orthogonal=True,
+    orthogonal_eps=1e-4,  # Increase epsilon
+)
+```
+
+### Problem: Limited Adaptation
+**Solution:**
+```python
+# Increase adaptation capacity
+config = BoneConfig(
+    bottleneck_size=128,  # Larger bottleneck
+    target_modules=["attention.output", "intermediate"],  # More target modules
+    dropout=0.1,
+    use_orthogonal=True,
+)
+```
+
+## Examples
```

Comment:
I don't think we need a section for "Examples" and also "Basic Usage". As mentioned above, let's link to the existing examples instead. If those are insufficient, I would much rather like to see those examples extended, e.g. to show a text classification task.